### DISCUSSION TITLE
Long string in Markdown exceeds style layout

### DISCUSSION THREAD

**Discussion by w1ndys** [squidfunk/mkdocs-material#7264] [Q&A] [answered]
I found that long text strings in Markdown, such as hash values, flag values, etc., will exceed the layout box. Is this because the default word is displayed on a single line?

![Screenshot_2024-06-11-20-40-11-216_com android chrome-edit](https://github.com/squidfunk/mkdocs-material/assets/109416673/827e6060-81ab-402c-8b5c-c29abb9290dc)
![Screenshot_2024-06-11-20-37-18-830_com tencent mm-edit](https://github.com/squidfunk/mkdocs-material/assets/109416673/b1044636-18dc-4b63-951b-ccd0ce2ed081)

**Comment by alexvoss** [comment, marked as answer]
You could try and play with the CSS [word-break](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break) property if you want to make these wrap around.
